ambry-store/src/main/java/com.github.ambry.store/BlobStoreStats.java

```diff
@@ -378,7 +378,7 @@ public void close() {
         referenceTimeInMs);
     long startTimeMs = time.milliseconds();
     Map<StoreKey, Long> deletedKeys = new HashMap<>();
-    NavigableMap<String, Long> validSizePerLogSegment = new TreeMap<>();
+    NavigableMap<String, Long> validSizePerLogSegment = new TreeMap<>(LogSegmentNameHelper.COMPARATOR);
```

Comment:
I had created an issue and assigned it to you when the bug was found (#713). We can use the same one to track tests.